meal_planner/form_processing.py

```
@@ -1,10 +1,41 @@
 from starlette.datastructures import FormData
+from typing import Optional
+
+from pydantic import BaseModel, Field, HttpUrl
+
+from meal_planner.models import RecipeBase
 
 
 def parse_recipe_form_data(form_data: FormData, prefix: str = "") -> dict:
-    """Parses recipe form data into a dictionary, handling optional prefix."""
+    """Parse recipe form data from multipart form submission.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)